Cogs/misc.py

```diff
@@ -52,34 +53,39 @@ async def ping(self, ctx):
                 latencyList.append(self.bot.latency)
 
             embed.add_field(name = "Average", value=f"{round((sum(latencyList)/tests) * 1000, 2)} ms", inline = False)
-            await ctx.send(embed = embed)
+            await ctx.respond(embed = embed)
         except Exception as e:
             print(e)
 ###########################################################################
-    @commands.command(name = "purge", aliases = ["clear", "delete"]) #Clears previous x amount of messages (x between 1 & 50)
+    @discord.slash_command(name = "purge", debug_guilds=[887366492730036276]) #Clears previous x amount of messages (x between 1 & 50)
     @commands.has_guild_permissions(manage_messages = True)
-    async def purge(self, ctx, limit: int):
+    async def purge(self, ctx, limit: discord.Option(discord.SlashCommandOptionType.integer, "Amount of messages to clear (1-50)", required = True)):
+        # Currently disabled b/c ctx.message.channel no longer exists and can't find alternative

```

Comment:
Because its no longer a message being sent. should be able to use context.channel though no?

Reply:
You're right, not sure why I didn't think of this before. Fixed it now.